gradlew.bat

```diff
@@ -0,0 +1,89 @@
+@rem

```

Comment:
Do we need to commit the Windows version? (I'd argue we shouldn't commit the Unix one either, but I've lost that battle a long time ago)

Reply:
It's in the other repos but as I don't think anyone has managed to get DECS running on Windows successfully it is basically just clutter for completeness so I'll get rid